### PR TITLE
Remove some older Jackson 2.x versions

### PR DESCRIPTION
**Pull request by tevans78**
#build

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.jaxrs.2.0_fat/build.gradle [hunk 1]
```diff
@@ -46,17 +46,17 @@ dependencies {
     'org.codehaus.jackson:jackson-jaxrs:1.6.2',
     'org.codehaus.jackson:jackson-mapper-asl:1.6.2',
     'org.codehaus.jackson:jackson-xc:1.6.2'
-  jackson2x 'com.fasterxml.jackson.core:jackson-annotations:2.4.1',
-    'com.fasterxml.jackson.core:jackson-core:2.4.1',
-    'com.fasterxml.jackson.core:jackson-databind:2.4.1',
-    'com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.4.1',
-    'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.4.1',
-    'com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.4.1'
+  jackson2x 'com.fasterxml.jackson.core:jackson-annotations:2.11.2',
```
Comment (cbridgha) [inline dev/com.ibm.ws.jaxrs.2.0_fat/build.gradle:49]: would it be possible to replace all the individual jackson lib references with the one or two wrapper projects?  This way - any update in the future is in one place...   This goes for all gradle file refs.

would look like:  project(':io.openliberty.com.fasterxml.jackson'),

**Review by cbridgha — CHANGES_REQUESTED** [review]
These changes look great - and happy to see a much more recent version being used..   suggested for a project ref if possible in the gradle files to reduce direct lib references

File: dev/com.ibm.ws.jaxrs.2.0_fat/build.gradle (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.ws.jaxrs.2.0_fat/build.gradle]: done

**Comment by tevans78** [comment]
#build 
#run-libby-bot

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Review by cbridgha — APPROVED** [review]
Thanks looks great

**Comment by tevans78** [comment]
#run-libby-bot

**Pull request merged**